src/TitleDetails/AddPieceModal/PieceFields/PieceFields.js

```diff
@@ -205,22 +223,23 @@ export const PieceFields = ({
               />
             </Col>
 
-            {isDiscoverySuppressEnabled && (
-              <Col
-                xs={6}
-                md={3}
-              >
-                <Field
-                  component={Checkbox}
-                  disabled={!values.displayOnHolding}
-                  fullWidth
-                  label={<FormattedMessage id="ui-receiving.piece.discoverySuppress" />}
-                  name="discoverySuppress"
-                  type="checkbox"
-                  vertical
-                />
-              </Col>
-            )}
+            {
+              showDisplayToPublic && (
```

Comment:
please name identifiers in the same way as is already done in other places `isDisplay...`